### DISCUSSION TITLE
MAUI and Shiny

### DISCUSSION THREAD

**Discussion by powerteqtpitman** [shinyorg/shiny#1021] [Q&A] [answered]
I just installed MAUI on my Mac with the Visual Studio Preview 3.

When I try to install the Shiny Core and Shiny bluetooth I get a package conflict error that MAUI requires AndroidX 1.7.0.2 and Shiny requires <= 1.7.0.

Is there an ETA on resolving this? I do not know much about how nuget versioning works. Is this an easy change to Shiny to allow 1.7.0.2? Are there big differences in the .2 release that cause issues with Shiny?

**Comment by aritchie** [comment, marked as answer]
I'm not discussing MAUI at this time as the tooling from them is not at a stable point.  Version 3 of Shiny also isn't at a stable point yet to start accepting issues.  It's why it is in alpha

With regards to ETA, no... this is not a product.  When I have time and the tooling works, I will get it done.

**Reply by powerteqtpitman** [reply]
Thank you so much for the quick response. I will await the final release of things before trying to us it.
